src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -30,7 +30,7 @@ import scala.reflect.ClassTag
   *  @define willNotTerminateInf
   */
 @SerialVersionUID(1L)
-class ArrayDeque[A] private(var array: Array[AnyRef], var start: Int, var end: Int)
+class ArrayDeque[A] private(var array: Array[Any], var start: Int, var end: Int)
```

Comment:
Indeed.  The speed/memory tradeoff here has never been worth it in any other collection.  I would be really surprised if it was this time.